File: backend/src/main/java/com/bluebee/smartsupply/model/Vechile.java
package com.bluebee.smartsupply.model;

import java.io.Serializable;

public class Vechile implements Serializable {

    private Integer vechilecd;
    private String vechileregno;
    private String ownername;
    private String addess1;
    private String address2;
    private String address3;
    private String zipcode;
    private String mobileno;
    private Integer appusercd;
    private Integer status;
     
     public Integer getVechilecd(){ 
    return this.vechilecd;
    }
    public void setVechilecd(Integer vechilecd){ 
    this.vechilecd=vechilecd;
    }
    public String getVechileregno(){ 
    return this.vechileregno;
    }
    public void setVechileregno(String vechileregno){ 
    this.vechileregno=vechileregno;
    }
    public String getOwnername(){ 
    return this.ownername;
    }
    public void setOwnername(String ownername){ 
    this.ownername=ownername;
    }
    public String getAddess1(){ 
    return this.addess1;
    }
    public void setAddess1(String addess1){ 
    this.addess1=addess1;
    }
    public String getAddress2(){ 
    return this.address2;
    }
    public void setAddress2(String address2){ 
    this.address2=address2;
    }
    public String getAddress3(){ 
    return this.address3;
    }
    public void setAddress3(String address3){ 
    this.address3=address3;
    }
    public String getZipcode(){ 
    return this.zipcode;
    }
    public void setZipcode(String zipcode){ 
    this.zipcode=zipcode;
    }
    public String getMobileno(){ 
    return this.mobileno;
    }
    public void setMobileno(String mobileno){ 
    this.mobileno=mobileno;
    }
    public Integer getAppusercd(){ 
    return this.appusercd;
    }
    public void setAppusercd(Integer appusercd){ 
    this.appusercd=appusercd;
    }
    public Integer getStatus(){ 
    return this.status;
    }
    public void setStatus(Integer status){ 
    this.status=status;
    }


}

File: frontend/app/src/main/java/com/asuravan/smartsupply/pojo/Vechile.kt
package com.asuravan.smartsupply.pojo

import android.os.Parcel
import android.os.Parcelable

class Vechile() :Parcelable {

     var vechilecd: Int? = null
     var vechileregno: String? = null
     var ownername: String? = null
     var addess1: String? = null
     var address2: String? = null
     var address3: String? = null
     var zipcode: String? = null
     var mobileno: String? = null
     var appusercd: Int? = null
     var status: Int? = null

    constructor(parcel: Parcel) : this() {
        vechilecd = parcel.readValue(Int::class.java.classLoader) as? Int
        vechileregno = parcel.readString()
        ownername = parcel.readString()
        addess1 = parcel.readString()
        address2 = parcel.readString()
        address3 = parcel.readString()
        zipcode = parcel.readString()
        mobileno = parcel.readString()
        appusercd = parcel.readValue(Int::class.java.classLoader) as? Int
        status = parcel.readValue(Int::class.java.classLoader) as? Int
    }

    override fun writeToParcel(parcel: Parcel, flags: Int) {
        parcel.writeValue(vechilecd)
        parcel.writeString(vechileregno)
        parcel.writeString(ownername)
        parcel.writeString(addess1)
        parcel.writeString(address2)
        parcel.writeString(address3)
        parcel.writeString(zipcode)
        parcel.writeString(mobileno)
        parcel.writeValue(appusercd)
        parcel.writeValue(status)
    }

    override fun describeContents(): Int {
        return 0
    }

    companion object CREATOR : Parcelable.Creator<Vechile> {
        override fun createFromParcel(parcel: Parcel): Vechile {
            return Vechile(parcel)
        }

        override fun newArray(size: Int): Array<Vechile?> {
            return arrayOfNulls(size)
        }
    }


}